unsupervised_rock_detection_3d/README.md

```diff
@@ -4,10 +4,24 @@
 
 This module extends the 3D Rock Detection project by focusing on unsupervised segmentation techniques for 3D rock point clouds. It emphasizes k-means clustering and region growing segmentation to enhance rock detection accuracy. The k-means clustering technique groups similar points based on their spatial features, while region growing relies on seed points and neighboring criteria to expand regions. These techniques enable precise segmentation of rock points from supporting surfaces, thus facilitating in-depth geometric analysis of geological formations.
 
+To enhance usability and enable interactive feedback, we have also developed a Graphical User Interface (GUI). The GUI allows geologists and students to run segmentation algorithms, visualize each step, and provide feedback by selecting initial seeds or basal points.
```

Comment:
feedback may cause some confusion, e.g., education feedback, feedback control, 

we can change it to "control"